index.rst

```diff
@@ -1,51 +1,628 @@
-..
-  Technote content.
+.. vim: ts=3:sts=3
 
-  See https://developer.lsst.io/docs/rst_styleguide.html
-  for a guide to reStructuredText writing.
+:tocdepth: 1
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+.. note::
 
-  Use the following syntax for sections:
+   This material was transcribed from the `Winter 2014 CameraGeom Design`_, originally written by Russell Owen & Simon Krughoff.
+   See also material on `the representation of a camera`_ and the `CameraGeom documentation`_.
 
-  Sections
-  ========
+.. _Winter 2014 CameraGeom Design: https://dev.lsstcorp.org/trac/wiki/Winter2014/Design/CameraGeomDesign
```

Comment:
Oh. Serves me right for not reading further in the document. No, if this DMTN is that trac page ported over then that makes me very happy. Sorry. If that's the case can you make sure that the Trac page has a banner on it saying that the content has moved?